### ISSUE TITLE
After installing the latest version 1.1.0, it keeps on STARTING!

### ISSUE THREAD

**Issue by iamjinlei0312** [langflow-ai/langflow#4588]
### Bug Description

![image](https://github.com/user-attachments/assets/b34abd49-b645-4f8a-bcc1-774a675f0a32)

I simply upgraded to the latest version with the command pip install -U langflow, but when I run python -m langflow run, I keep getting the above screen.

### Reproduction

To run langflow, I started a separate virtual environment.

python version is 3.11.9.

pip list 

```
pip list
Package                                  Version
---------------------------------------- ---------------
aiofiles                                 24.1.0         
aiohappyeyeballs                         2.4.3          
aiohttp                                  3.9.5          
aiolimiter                               1.1.0          
aiosignal                                1.3.1          
aiosqlite                                0.20.0         
alembic                                  1.13.3         
annotated-types                          0.7.0          
anthropic                                0.39.0         
anyio                                    4.6.2.post1    
appdirs                                  1.4.4          
asgiref                                  3.8.1          
assemblyai                               0.34.0         
astra-assistants                         2.2.6
astrapy                                  1.5.2
asttokens                                2.4.1
asyncer                                  0.0.8
atlassian-python-api                     3.41.16
attrs                                    23.2.0
auth0-python                             4.7.2
Authlib                                  1.3.1
backoff                                  2.2.1
backports.tarfile                        1.2.0
bce-python-sdk                           0.9.23
bcrypt                                   4.0.1
beautifulsoup4                           4.12.3
boto3                                    1.34.162
botocore                                 1.34.162
build                                    1.2.2.post1
CacheControl                             0.14.0
cachetools                               5.5.0
cassandra-driver                         3.29.2
cassio                                   0.1.10
cattrs                                   24.1.2
certifi                                  2024.8.30
cffi                                     1.17.1
chardet                                  5.2.0
charset-normalizer                       3.4.0
chroma-hnswlib                           0.7.6
chromadb                                 0.5.18
cleo                                     2.1.0
clevercsv                                0.8.2
click                                    8.1.7
clickhouse-connect                       0.7.19
cohere                                   5.11.1
colorama                                 0.4.6
coloredlogs                              15.0.1
colorlog                                 6.8.2
composio_core                            0.5.9
composio_langchain                       0.5.9
coolname                                 2.2.0
crashtest                                0.4.1
crewai                                   0.79.4
crewai-tools                             0.14.0
cryptography                             43.0.3
dataclasses-json                         0.6.7
datasets                                 2.2.1
decorator                                5.1.1
deepdiff                                 8.0.1
defusedxml                               0.7.1
Deprecated                               1.2.14
deprecation                              2.1.0
dill                                     0.3.9
diskcache                                5.6.3
distlib                                  0.3.9
distro                                   1.9.0
dnspython                                2.7.0
docker                                   7.1.0
docstring_parser                         0.16
docx2txt                                 0.8
dspy                                     2.5.15
dspy-ai                                  2.5.15
duckdb                                   1.1.2
duckduckgo_search                        6.3.4
dulwich                                  0.21.7
durationpy                               0.9
e2b                                      1.0.1
e2b-code-interpreter                     1.0.1
ecdsa                                    0.19.0
elastic-transport                        8.15.1
elasticsearch                            8.15.1
embedchain                               0.1.125
emoji                                    2.14.0
environs                                 9.5.0
Events                                   0.5
executing                                2.1.0
faiss-cpu                                1.9.0
fake-useragent                           1.5.1
fastapi                                  0.115.2
fastapi-pagination                       0.12.31
fastavro                                 1.9.7
fastjsonschema                           2.20.0
filelock                                 3.16.1
firecrawl-py                             1.5.0
FLAML                                    2.3.1
flatbuffers                              24.3.25
frozendict                               2.4.6
frozenlist                               1.4.1
fsspec                                   2024.10.0
future                                   1.0.0
geomet                                   0.2.1.post1
gitdb                                    4.0.11
GitPython                                3.1.43
google-ai-generativelanguage             0.6.10
google-api-core                          2.21.0
google-api-python-client                 2.149.0
google-auth                              2.35.0
google-auth-httplib2                     0.2.0
google-auth-oauthlib                     1.2.1
google-cloud-aiplatform                  1.70.0
google-cloud-bigquery                    3.26.0
google-cloud-core                        2.4.1
google-cloud-resource-manager            1.12.5
google-cloud-storage                     2.18.2
google-crc32c                            1.6.0
google-generativeai                      0.8.3
google-resumable-media                   2.7.2
google_search_results                    2.4.2
googleapis-common-protos                 1.65.0
gotrue                                   2.9.3
gptcache                                 0.1.44
gql                                      3.5.0
grandalf                                 0.8
graphql-core                             3.2.5
greenlet                                 3.1.1
groq                                     0.11.0
grpc-google-iam-v1                       0.13.1
grpcio                                   1.67.0
grpcio-health-checking                   1.62.3
grpcio-status                            1.62.3
grpcio-tools                             1.62.3
gunicorn                                 23.0.0
h11                                      0.14.0
h2                                       4.1.0
hpack                                    4.0.0
html5lib                                 1.1
httpcore                                 1.0.6
httplib2                                 0.22.0
httptools                                0.6.4
httpx                                    0.27.0
httpx-sse                                0.4.0
huggingface-hub                          0.26.1
humanfriendly                            10.0
hyperframe                               6.0.1
idna                                     3.10
ijson                                    3.3.0
importlib_metadata                       8.4.0
importlib_resources                      6.4.5
inflection                               0.5.1
iniconfig                                2.0.0
installer                                0.7.0
instructor                               1.3.3
ipython                                  8.28.0
jaraco.classes                           3.4.0
jaraco.context                           6.0.1
jedi                                     0.19.1
Jinja2                                   3.1.4
jiter                                    0.4.2
jmespath                                 1.0.1
joblib                                   1.4.2
jq                                       1.8.0
json_repair                              0.25.3
jsonpatch                                1.33
jsonpath-python                          1.0.6
jsonpickle                               3.3.0
jsonpointer                              3.0.0
jsonref                                  1.1.0
jsonschema                               4.23.0
jsonschema-specifications                2024.10.1
keyring                                  24.3.1
kubernetes                               31.0.0
lancedb                                  0.15.0
langchain                                0.3.7
langchain-anthropic                      0.3.0
langchain-astradb                        0.5.2
langchain-aws                            0.2.7
langchain-chroma                         0.1.4
langchain-cohere                         0.3.1
langchain-community                      0.3.4
langchain-core                           0.3.18
langchain-elasticsearch                  0.3.0
langchain-experimental                   0.3.3
langchain-google-calendar-tools          0.0.1
langchain-google-community               2.0.2
langchain-google-genai                   2.0.4
langchain-google-vertexai                2.0.7
langchain-groq                           0.2.1
langchain-milvus                         0.1.6
langchain-mistralai                      0.2.1
langchain-mongodb                        0.2.0
langchain-nvidia-ai-endpoints            0.3.5
langchain-ollama                         0.2.0
langchain-openai                         0.2.8
langchain-pinecone                       0.2.0
langchain-text-splitters                 0.3.2
langchain-unstructured                   0.1.5
langchainhub                             0.1.21
langflow                                 1.1.0
langflow-base                            0.1.0
langfuse                                 2.53.9
langsmith                                0.1.136
langwatch                                0.1.16
lark                                     1.2.2
litellm                                  1.49.1
loguru                                   0.7.2
lsprotocol                               2023.0.1
lxml                                     5.3.0
lz4                                      4.3.3
magicattr                                0.1.6
Mako                                     1.3.6
Markdown                                 3.7
markdown-it-py                           3.0.0
MarkupSafe                               3.0.2
marshmallow                              3.23.0
matplotlib-inline                        0.1.7
mdurl                                    0.1.2
mem0ai                                   0.1.29
metal_sdk                                2.5.1
metaphor-python                          0.1.23
mmh3                                     5.0.1
monotonic                                1.6
more-itertools                           10.5.0
mpmath                                   1.3.0
msgpack                                  1.1.0
multidict                                6.1.0
multiprocess                             0.70.17
multitasking                             0.0.11
mypy-extensions                          1.0.0
nanoid                                   2.0.0
nest-asyncio                             1.6.0
networkx                                 3.4.2
nltk                                     3.9.1
nodeenv                                  1.9.1
numexpr                                  2.10.1
numpy                                    1.26.4
oauthlib                                 3.2.2
ollama                                   0.3.3
onnxruntime                              1.19.2
openai                                   1.54.4
opensearch-py                            2.7.1
opentelemetry-api                        1.27.0
opentelemetry-exporter-otlp-proto-common 1.27.0
opentelemetry-exporter-otlp-proto-grpc   1.27.0
opentelemetry-exporter-otlp-proto-http   1.27.0
opentelemetry-exporter-prometheus        0.48b0
opentelemetry-instrumentation            0.48b0
opentelemetry-instrumentation-asgi       0.48b0
opentelemetry-instrumentation-fastapi    0.48b0
opentelemetry-proto                      1.27.0
opentelemetry-sdk                        1.27.0
opentelemetry-semantic-conventions       0.48b0
opentelemetry-util-http                  0.48b0
optuna                                   4.0.0
orderly-set                              5.2.2
orjson                                   3.10.0
outcome                                  1.3.0.post0
overrides                                7.7.0
packaging                                24.1
pandas                                   2.2.2
parameterized                            0.9.0
paramiko                                 3.5.0
parso                                    0.8.4
passlib                                  1.7.4
peewee                                   3.17.7
pexpect                                  4.9.0
pgvector                                 0.3.5
pillow                                   10.4.0
pinecone-client                          5.0.1
pinecone-plugin-inference                1.1.0
pinecone-plugin-interface                0.0.7
pip                                      24.2
pkginfo                                  1.11.2
platformdirs                             4.3.6
pluggy                                   1.5.0
poetry                                   1.8.4
poetry-core                              1.9.1
poetry-plugin-export                     1.8.0
portalocker                              2.10.1
postgrest                                0.16.11
posthog                                  3.7.0
primp                                    0.7.0
prometheus_client                        0.21.0
prompt_toolkit                           3.0.48
propcache                                0.2.0
proto-plus                               1.24.0
protobuf                                 4.25.5
psycopg                                  3.2.3
psycopg-binary                           3.2.3
psycopg2-binary                          2.9.10
ptyprocess                               0.7.0
pulsar-client                            3.5.0
pure_eval                                0.2.3
py                                       1.11.0
pyarrow                                  17.0.0
pyasn1                                   0.6.1
pyasn1_modules                           0.4.1
pyautogen                                0.3.1
pycparser                                2.22
pycryptodome                             3.21.0
pydantic                                 2.9.2
pydantic_core                            2.23.4
pydantic-settings                        2.4.0
Pygments                                 2.18.0
PyJWT                                    2.9.0
pylance                                  0.19.1
pymilvus                                 2.4.8
pymongo                                  4.10.1
PyNaCl                                   1.5.0
pyparsing                                3.2.0
pypdf                                    5.1.0
pyperclip                                1.9.0
PyPika                                   0.48.9
pyproject_hooks                          1.2.0
pyreadline3                              3.5.4
pyright                                  1.1.389
pysbd                                    0.3.4
Pysher                                   1.0.8
PySocks                                  1.7.1
pytest                                   8.3.3
python-dateutil                          2.9.0.post0
python-docx                              1.1.2
python-dotenv                            1.0.1
python-jose                              3.3.0
python-lsp-jsonrpc                       1.1.2
python-multipart                         0.0.12
pytube                                   15.0.0
pytz                                     2024.2
pyvis                                    0.3.2
pywin32                                  308
pywin32-ctypes                           0.2.3
PyYAML                                   6.0.2
qdrant-client                            1.9.2
qianfan                                  0.3.5
ragstack-ai-knowledge-store              0.2.1
RapidFuzz                                3.10.0
realtime                                 1.0.6
redis                                    5.1.1
referencing                              0.35.1
regex                                    2024.9.11
requests                                 2.32.3
requests-oauthlib                        2.0.0
requests-toolbelt                        1.0.0
responses                                0.18.0
retry                                    0.9.2
rich                                     13.9.2
rpds-py                                  0.20.0
rsa                                      4.9
ruff                                     0.6.9
s3transfer                               0.10.3
schema                                   0.7.7
selenium                                 4.26.1
semver                                   3.0.2
sentry-sdk                               2.17.0
setuptools                               75.2.0
shapely                                  2.0.6
shellingham                              1.5.4
simsimd                                  5.8.0
six                                      1.16.0
smmap                                    5.0.1
sniffio                                  1.3.1
sortedcontainers                         2.4.0
soupsieve                                2.6
spider-client                            0.0.72
SQLAlchemy                               2.0.36
sqlmodel                                 0.0.18
stack-data                               0.6.3
starlette                                0.40.0
storage3                                 0.7.7
StrEnum                                  0.4.15
structlog                                24.4.0
supabase                                 2.6.0
supafunc                                 0.5.1
sympy                                    1.13.3
tabulate                                 0.9.0
tenacity                                 8.3.0
termcolor                                2.5.0
tiktoken                                 0.7.0
tokenizers                               0.20.1
toml                                     0.10.2
tomli                                    2.1.0
tomli_w                                  1.1.0
tomlkit                                  0.13.2
tqdm                                     4.66.5
traitlets                                5.14.3
tree-sitter                              0.23.1
tree-sitter-python                       0.23.2
trio                                     0.27.0
trio-websocket                           0.11.1
trove-classifiers                        2024.10.21.16
typer                                    0.13.0
types-cachetools                         5.5.0.20240820
types-requests                           2.32.0.20241016
typing_extensions                        4.12.2
typing-inspect                           0.9.0
tzdata                                   2024.2
ujson                                    5.10.0
uncurl                                   0.0.11
unstructured-client                      0.25.9
upstash-vector                           0.6.0
uritemplate                              4.1.1
urllib3                                  2.2.3
uuid6                                    2024.7.10
uv                                       0.5.1
uvicorn                                  0.32.0
validators                               0.34.0
virtualenv                               20.27.0
watchfiles                               0.24.0
wcwidth                                  0.2.13
weaviate-client                          4.9.0
webencodings                             0.5.1
websocket-client                         1.8.0
websockets                               12.0
wheel                                    0.38.4
wikipedia                                1.4.0
win32-setctime                           1.1.0
wolframalpha                             5.1.3
wrapt                                    1.16.0
wsproto                                  1.2.0
xmltodict                                0.14.2
xxhash                                   3.5.0
yarl                                     1.16.0
yfinance                                 0.2.46
youtube-transcript-api                   0.6.2
zep-python                               2.0.2
zipp                                     3.20.2
zstandard                                0.23.0
```
 I am running it on windos, not on my mac.

### Expected behavior

t should be possible to try the new version of the feature.

### Operating System

Windows 10 专业版

### Langflow Version

1.1.0

### Python Version

3.11

### Screenshot

![image](https://github.com/user-attachments/assets/b2c26417-4461-4f6c-8f57-7a2f1aef9743)

**Comment by italojohnny** [comment]
Hello, how are you?

I sincerely apologize for the delay in responding. During this time, we released a new version, 1.1.1.
Could you please check if the issue still occurs in this new version?

I’ve been trying to reproduce your problem but haven’t had any success. Could you try running Langflow with the following parameter?
```
python -m langflow run --log-level=debug
```
This will return more detailed logs, which I hope will shed some light on the issue.

**Comment by carlosrcoelho** [comment]
Thank you for your contribution! This issue will be closed. If you have any questions or encounter another problem, please open a new issue and we will be ready to help you.
